pilot/pkg/networking/plugin/authz/rbac_test.go

```diff
@@ -145,35 +145,31 @@ func TestBuildHTTPFilter(t *testing.T) {
 	}
 	store := newIstioStoreWithConfigs([]model.Config{roleCfg, roleCfgWithoutBinding, bindingCfg}, t)
 	testCases := []struct {
-		Name      string
-		Namespace string
-		Host      string
-		Store     model.IstioConfigStore
-		Policy    string
+		Name    string
+		Service *serviceMetadata
+		Store   model.IstioConfigStore
+		Policy  string
 	}{
 		{
-			Name:      "no matched role",
-			Namespace: "xyz",
-			Host:      "abc.xyz",
-			Store:     store,
+			Name:    "no matched role",
+			Service: &serviceMetadata{name: "abc.xyz"},
+			Store:   store,
 		},
 		{
-			Name:      "no matched binding",
-			Namespace: "default",
-			Host:      "review.default",
-			Store:     store,
+			Name:    "no matched binding",
+			Service: &serviceMetadata{name: "review.default"},
+			Store:   store,
 		},
 		{
-			Name:      "role with binding",
-			Namespace: "default",
-			Host:      "product.default",
-			Store:     store,
-			Policy:    "test-role-1",
+			Name:    "role with binding",
+			Service: &serviceMetadata{name: "product.default"},
+			Store:   store,
+			Policy:  "test-role-1",
 		},
 	}
 
 	for _, tc := range testCases {
-		filter := buildHTTPFilter(tc.Host, tc.Namespace, tc.Store)
+		filter := buildHTTPFilter(tc.Service, tc.Store)
 		if fn := "envoy.filters.http.rbac"; filter.Name != fn {
```

Comment:
replace "envoy.filters.http.rbac" with constant.

Reply:
I prefer to leave it as a constant in the test, just in case we don't accidentally change the value of `rbacFilterName`.